test/features/extensions/upgrade/upgrade.go

```diff
@@ -38,7 +38,7 @@ func Feature(t *testing.T) features.Feature {
 
 	// update to snapshot
 	withCSI := false
-	builder.Assess("upgrade operator", helpers.ToFeatureFunc(operator.InstallViaMake(withCSI), true))
+	builder.Assess("upgrade operator", helpers.ToFeatureFunc(operator.Install("", withCSI), true))
```

Comment:
such hacks hard to read, can we make more readable wrapper, aka InstallWithoutTag?